akka-docs/rst/java/code/jdocs/actor/TypedActorDocTest.java

```diff
@@ -198,7 +196,7 @@ public int squareNow(int i) {
     }
   }
 
-  //#typed-router-types-1
+  //#typed-router-types
```

Comment:
essentially yes - it would've worked if we used `-1` again below and at the inclusion point, but just  removing it seemed simpler ;)